test_values.go

```diff
@@ -17,4 +19,16 @@ var testGateway = New(
 	os.Getenv("BRAINTREE_PRIV_KEY"),
 )
 
+var testTimeZone = func() *time.Location {
+	tzName := os.Getenv("BRAINTREE_TIMEZONE")
+	if tzName == "" {
+		tzName = "US/Mountain"
```

Comment:
We could default it to UTC, and set the variable to `US/Mountain` in the `.travis.yml`?